src/app/(app)/layout.tsx

```
@@ -3,17 +3,20 @@
 import Header from '@/components/header/header';
 import { Sidebar } from '@/components/sidebar/sidebar';
 import styled from '@emotion/styled';
+import { usePathname } from 'next/navigation';
 
 const layout = ({
   children,
 }: Readonly<{
   children: React.ReactNode;
 }>) => {
+  const pathname = usePathname();
```

ESLint

[error] 'pathname' is assigned a value but never used.

(`@typescript-eslint/no-unused-vars`)

---

[error] React Hook "usePathname" is called in function "layout" that is neither a React function component nor a custom React Hook function. React component names must start with an uppercase letter. React Hook names must start with the word "use".

(react-hooks/rules-of-hooks)